tools/distributions/distribution_rules.bzl

```diff
@@ -14,11 +14,11 @@
 
 load("@rules_java//java:defs.bzl", _java_import = "java_import")
 
-def distrib_java_import(name, enable_distributions = [], **kwargs):
+def distrib_java_import(name, visibility = None, enable_distributions = [], **kwargs):
```

Comment:
please split this off into a separate commit

Reply:
Done, I sent another CL to fix the visibility problem.